fedimintd/src/lib.rs

```diff
@@ -2,31 +2,33 @@ use bitcoin::Network;
 use fedimint_core::config::{ConfigGenParams, ModuleGenRegistry};
 use fedimint_core::module::DynModuleGen;
 use fedimint_core::{Amount, Tiered};
-use fedimint_ln::LightningGen;
+use fedimint_ln::{LightningGen, LightningGenParams};
 use fedimint_mint::{MintGen, MintGenParams};
 use fedimint_wallet::{WalletGen, WalletGenParams};
 
 pub mod ui;
 
 /// Generates the configuration for the modules configured in the server binary
+// FIXME: Currently the client depends on this exact order being configured
 pub fn configure_modules(
     max_denomination: Amount,
     network: Network,
     finality_delay: u32,
 ) -> ConfigGenParams {
     ConfigGenParams::new()
-        .attach(WalletGenParams {
-            network,
-            // TODO this is not very elegant, but I'm planning to get rid of it in a next commit
-            // anyway
-            finality_delay,
-        })
+        .attach(LightningGenParams)
         .attach(MintGenParams {
             mint_amounts: Tiered::gen_denominations(max_denomination)
                 .tiers()
                 .cloned()
                 .collect(),
         })
+        .attach(WalletGenParams {
+            network,
+            // TODO this is not very elegant, but I'm planning to get rid of it in a next commit
```

Comment:
BTW. "the next commit" did not materialize. :D

Reply:
Yeah what was the context on the comment?